openmetadata-service/src/main/java/org/openmetadata/csv/CsvUtil.java

```diff
@@ -13,9 +13,7 @@
 
 package org.openmetadata.csv;
 
-import static org.openmetadata.common.utils.CommonUtil.listOf;
-import static org.openmetadata.common.utils.CommonUtil.listOrEmpty;
-import static org.openmetadata.common.utils.CommonUtil.nullOrEmpty;
+import static org.openmetadata.common.utils.CommonUtil.*;
```

Comment:
please do not use wild-card imports. can you keep the old code here and update the PR

Reply:
code updated..